### PR TITLE
[Adjustment][Sam] Footer UI & Navigation

### PR DESCRIPTION
**Pull request by franzsamilo**
**Clickup Task:**
https://app.clickup.com/t/865cqywum
https://app.clickup.com/t/865cqyx6f

**Description:**
- Fixed Footer nav's alignment
- Fixed Navigations
- Removed random padding above nav in home screen

- Nav Icon changes according to active screen (not yet tested since user screen is not yet merged)

**Images/Videos:**

https://github.com/ezerssss/unihub/assets/124412484/3caee6ec-6986-4cd1-973b-8200e1899b94

### CODE REVIEW

**Review by HansGabriel — APPROVED** [review]

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -18,36 +19,52 @@ function Footer() {
   const navigation =
     useNavigation<NativeStackNavigationProp<RootStackParamsList>>();
 
+  const [currentScreen, setCurrentScreen] = useState(Routes.HOME);
```
Comment (ezerssss) [inline components/Footer/Footer.tsx:22]: no need for `useState` i think kay u can use `useRoute` and it should give the current screen

image

you can follow how i implemented it sa `useAuthHandler` na file

`route.name`

**Review by ezerssss — CHANGES_REQUESTED** [review]

**Review by ezerssss — APPROVED** [review]

**Review by jedmamosto — CHANGES_REQUESTED** [review]
`Sell` doesn't seem centered? It looks off and feels off. 

Add more space between the indicator bar and the Footer components - they look tight.

image

**Comment by jedmamosto** [comment]
Also I think before we can merge this, we need to have the User screen merged into main nadin?

**Review by jedmamosto — CHANGES_REQUESTED** [review]
Navigation works but... Home icon should change too

image

Should be: 

image


Also sell doesn't seem to look centered still... its a bit off to the left and margin to the right of `Me` seems to be more compared to `Shop`

**What it looks like:** 
image


Also add space between the navigation icons like: (not exactly 14.43 but like close to that so maybe 14px or 16px)

image

**Comment by caspted** [comment]
The circle part at bottom is my home screen button and that is in the center of the screen so that is where you will base to move the Sell towards the center

**Comment by jedmamosto** [comment]
`Sell` is still not centered... as well as it leans more on the left...

Here are spacing guidelines:
image
image
image
image

**_Always keep `Sell` at the center_**

**Review by jedmamosto — APPROVED** [review]

**Pull request merged**